src/routes/protected.tsx

```diff
@@ -326,6 +327,14 @@ export const protectedRoutes: RouteNode[] = [
               </ProjectEditRoute>
             ),
           },
+          {
+            path: ProjectDashboardRoutes.EXTERNAL_FORM_SUBMISSIONS,
+            element: (
+              <ProjectEditRoute>
```

Comment:
This will make it so the page automatically resolves `<NotFound />` if user lacks `canManageExternalFormSubmissions` for this project. 
```suggestion
              <ProjectEditRoute permissions={['canManageExternalFormSubmissions']}>
```